lib/usd/ui/layerEditor/layerTreeItem.h

```diff
@@ -46,19 +47,54 @@ enum class RebuildChildren
     No
 };
 
+enum LayerActionType
+{
+    None,
+    Mute,
+    Lock
+};
+
+enum LayerMasks
+{
+    LayerMasks_None = (1u << 0),     // 00000000
+    LayerMasks_Session = (1u << 1),  // 00000001
+    LayerMasks_Root = (1u << 2),     // 00000010
+    LayerMasks_SubLayer = (1u << 3), // 00000100
+};
+
+static LayerMasks CreateLayerMask(bool isRootLayer, bool isSubLayer, bool isSessionLayer)
```

Comment:
I removed the static and currently there hasn't been a need for mask level checks. I plan to create a bit mask class later so it can be inherited and used in multiple places. 
The purpose of the three parameter is to avoid using | , & and other variations at the target code.

For the time being, I removed static and moved the definition to the .cpp side.

Reply:
My comment was about that it is easy to get the order of three booleans wrong.